index.html

```diff
@@ -217,5 +217,53 @@ <h2 class="skill-name">Skills</h2>
 			</li>
 		</ul>
 	</section>
+	<section  class="contact-container" id="footer-container-id">
+		<div class="contact-form">
+			<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>		
```

Comment:
- [ ] It is important that you stick to the design templates in Figma as much as possible, this includes the font, colours. Kindly use the colour specified in Figma and not `black`.

Your Design | Figma Design
--|--
![image](https://user-images.githubusercontent.com/29071984/213036408-038afa85-7fef-4162-8feb-08f3574ae762.png) | ![image](https://user-images.githubusercontent.com/29071984/213036622-03cda039-fb0c-4851-aaef-b8d0377b8976.png)